sdk/tests/deqp/functional/gles3/es3fPixelBufferObjectTest.js

```diff
@@ -287,6 +287,9 @@ var tcuImageCompare = framework.common.tcuImageCompare;
                 break;
         }
 
+        // GL_DITHER will cause the dither on color component which effects the result of rendering.
```

Comment:
Could you please indicate in this comment that this is a difference compared to the native dEQP?